pom.xml

```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <version>2.2.4</version>
```

Comment:
Please don't change this - the release process bumps it automagically.

Reply:
changing this back

Reply:
Just wanted to make sure I should change this back. I think you may have thought I was bumping the version of this plugin, but I'm bumping the version of the git plugin dependency since newer versions support the url parameter in the RevisionParameterAction.

Reply:
Yea - even once the PR is merged, that value won't be used. It's only used when I push a real release.

Reply:
BTW - I thought you were changing the version of the plugin, not the git plugin. My bad, and that's why I merged ;-)